src/commands/dev.ts

```diff
@@ -378,6 +386,53 @@ export default class Dev extends BaseCommand {
     return port;
   }
 
+  private async downloadFile(url: string, output_location: string): Promise<void> {
+    const handleReject = (resolve: () => void, reject: () => void) => {
+      // These file operations can be sync due to failure state
+      if (!fs.existsSync(output_location)) {
+        return reject();
+      }
+      // If the file is not too old than we can use it instead
+      const stats = fs.statSync(output_location);
+      const diff_in_ms = Math.abs(stats.mtime.getTime() - Date.now());
+      const days = diff_in_ms / (1000 * 60 * 60 * 24);
+      if (days > 30) {
+        reject();
+      } else {
+        resolve();
+      }
+    };
+    return new Promise((resolve, reject) => {
+      axios({
+        method: 'get',
+        url: url,
+        responseType: 'stream',
+      }).then((response) => {
+        if (response.status > 399) {
+          return handleReject(resolve, reject);
+        }
+        const writer = createWriteStream(output_location);
+        response.data.pipe(writer);
+        response.data.on('end', resolve);
+        response.data.on('error', () => {
+          return handleReject(resolve, reject);
+        });
+      }).catch(err => {
+        return handleReject(resolve, reject);
+      });
+    });
+  }
+
+  private async downloadSSLCerts() {
+    return Promise.all([
+      this.downloadFile('https://storage.googleapis.com/architect-ci-ssl/fullchain.pem', path.join(this.app.config.getConfigDir(), 'fullchain.pem')),
```

Comment:
Does this work correctly on windows? Sometimes you need to call `path.resolve` to get the correct path for the os. Maybe path.join does the same thing.

Reply:
Just tested on Windows and it works as expected. From my understanding `path.join` is supposed to be platform agnostic.